### ISSUE TITLE
Failed to fetch document tree (401 Unauthorized Error)

### ISSUE THREAD

**Issue by djslzx** [juruen/rmapi#181]
Hi,

I've been using rmapi on Linux (Pop OS) with no issue for the past few months.  Recently, however, I've started to get the following error when running rmapi:
```
ERROR: 2021/05/27 07:09:32 main.go:42: failed to build documents tree, last error: failed to fetch document tree 401 Unauthorized Error
```

I've tried re-installing rmapi from source, using the newest release (v0.0.15), and resetting rmapi by deleting the config file in `~/.config/rmapi/rmapi.conf` to use a new device code.

1. When reinstalling from source and entering a new code, I get the same error as above.
2. When using the release and entering a new code, I get the following error: `ERROR: 2021/05/27 07:12:02 auth.go:93: failed to create a new device token`.  Then the process seems to continue running, eating up a ton of CPU; I had to manually kill it.
3. When deleting the config file to use a new code, the results are the same as (1).

I've also tried running the Windows desktop app, which worked fine, and the Linux desktop app (I installed it a while back, and I'm not sure how to update it), which had a similar error as rmapi.

I'd appreciate any help in figuring out what's going on.  Thanks!

**Comment by remiq** [comment]
Duplicate of solved issue #177

**Comment by djslzx** [comment]
Duplicate of #177
